protocol/x/clob/types/keys.go

```diff
@@ -18,94 +18,91 @@ const (
+	EquityTierLimitConfigKey = "EqTierLimCfg"
 
 	// BlockRateLimitConfigKey is the key to retrieve the block rate limit configuration.
-	BlockRateLimitConfigKey = "block_rate_limit_config"
+	BlockRateLimitConfigKey = "RateLimCfg"
 
 	// ClobPairKeyPrefix is the prefix to retrieve all ClobPair
-	ClobPairKeyPrefix = "clob_pair/"
+	ClobPairKeyPrefix = "Clob/"
 )
 
 const (
 	// OrderAmountFilledKeyPrefix is the prefix to retrieve the fill amount for an order.
-	OrderAmountFilledKeyPrefix = "order_amount_filled/"
+	OrderAmountFilledKeyPrefix = "Filled/"
 
 	// BlockHeightToPotentiallyPrunableOrdersPrefix is the prefix to retrieve a list of potentially prunable orders
 	// by block height.
-	BlockHeightToPotentiallyPrunableOrdersPrefix = "block_height_to_potentially_prunable_orders/"
-
-	// OrdersFilledDuringLatestBlockKey is the key to retrieve the list of orders filled during the latest block.
-	OrdersFilledDuringLatestBlockKey = "orders_filled_during_latest_block"
+	BlockHeightToPotentiallyPrunableOrdersPrefix = "ExpHt/"
 )
 
 // Below key prefixes are not explicitly used to read/write to state, but rather used to iterate over
 // certain groups of items stored in state.
 const (
 	// StatefulOrderKeyPrefix is the prefix key for all long term orders and all conditional orders,
 	// both triggered and untriggered.
-	StatefulOrderKeyPrefix = "stateful_order_placement/"
+	StatefulOrderKeyPrefix = "SOP/"
 
 	// PlacedStatefulOrderKeyPrefix is the prefix key for placed long term orders and triggered
 	// conditional orders. It represents all stateful orders that should be placed upon the memclob
 	// during app start up.
-	PlacedStatefulOrderKeyPrefix = StatefulOrderKeyPrefix + "placed/"
+	PlacedStatefulOrderKeyPrefix = StatefulOrderKeyPrefix + "P/"
 )
 
 // Store / Memstore
 const (
 	// TriggeredConditionalOrderKeyPrefix is the key to retrieve an triggered conditional order and
 	// information about when it was triggered.
-	TriggeredConditionalOrderKeyPrefix = PlacedStatefulOrderKeyPrefix + "conditional/"
+	TriggeredConditionalOrderKeyPrefix = PlacedStatefulOrderKeyPrefix + "C/"
 
 	// LongTermOrderPlacementKeyPrefix is the key to retrieve a long term order and information about
 	// when it was placed.
-	LongTermOrderPlacementKeyPrefix = PlacedStatefulOrderKeyPrefix + "long_term/"
+	LongTermOrderPlacementKeyPrefix = PlacedStatefulOrderKeyPrefix + "LT/"
 
 	// UntriggeredConditionalOrderKeyPrefix is the key to retrieve an untriggered conditional order and
 	// information about when it was placed.
-	UntriggeredConditionalOrderKeyPrefix = StatefulOrderKeyPrefix + "untriggered/conditional/"
+	UntriggeredConditionalOrderKeyPrefix = StatefulOrderKeyPrefix + "Untrig/"
 
 	// StatefulOrdersTimeSlicePrefix is the key to retrieve a unique list of the stateful orders that
 	// expire at a given timestamp, sorted by order ID.
-	StatefulOrdersTimeSlicePrefix = "stateful_orders_time_slice/"
+	StatefulOrdersTimeSlicePrefix = "ExpTime/"
 )
```

Comment:
these are probably the most important to shorten in state since there will be a lot.

Suggest:

`SO/`
`P/` (unchanged)
`T/` (triggered)
`L/` (longterm)
`U/` (untriggered)
`ExpTm/`